man/hptlink.1

```diff
@@ -3,7 +3,10 @@
 hptlink \- messages linker for hpt
 .SH SYNTAX
 .B hptlink
-[\-t] [\-s] [\-a] [\-r] [\-l loglevel] [\-n] [areaname ...]
+[\-c config] [\-t] [\-s] [\-a] [\-r] [\-l loglevel] [\-n] [areaname ...]
```

Comment:
I see in the code that "-l" does nothing. It would be better not to mention loglevel in this line and specify in the text below that the option does nothing, and it is here just for compatibility with the old versions.

Reply:
Done.